vignettes/implementing-cookies.Rmd

```diff
@@ -73,8 +73,10 @@ ui <- function(input, output, session) {
       id = "navlistPanel",
       widths = c(2, 8),
       well = FALSE,
-      dfeshiny::cookies_panel_ui(
-        google_analytics_key = google_analytics_key
+      shiny::tabPanel(
```

Comment:
One little quibble is that this (and `support_panel()`) doesn't align very well in the page. The container's too cramped up to the header when done like this. I get a nicer spacing when I bring the usual containers back in

```
gov_main_layout(
   gov_row(
      column(
         cookies_panel_ui(google_analytics_key = google_analytics_key)
         )
   )
)
```

Aware this is quite long-winded and you're not wanting to tie these functions to a specific package for the layout, but if we're not controlling that in the function itself, then I feel like we need something in the example at least to make sure people are getting a nice layout.

Reply:
I've had a play with this and think the spacing actually might be better without to give more space on the page for the dashboard, with your suggestion to me the content has too much space above it? The tab panel on the left does look too low in comparison, though I think that's where making our own navigation sooner rather than later might be worth doing and might make the spacing feel better - included screenshots below in case it's appearing any differently for you.

So for the time being I'm tempted not to add anything yet and instead update the examples once we have updated in the standard layout and have that agreed in the template first?

Example of wrapping the content in the panel like you suggest
![image](https://github.com/user-attachments/assets/777b5c24-8d8c-4dd7-97df-d2bf778ab293)

Example of wrapping the whole tabset in those containers (quickly realised this was even worse!)
![image](https://github.com/user-attachments/assets/9d1d9a23-0b74-43c1-95be-b110847b671f)

What I can see without adding anything
![image](https://github.com/user-attachments/assets/c4c5b044-9af3-459d-adc3-889a58c397e3)

Example of the provider dashboard with the prototype left nav
![image](https://github.com/user-attachments/assets/7a0e958f-b38c-4bcb-b6e2-d800cd7520f5)

Reply:
Huh, this is what I was getting in the template. Maybe there's a css or other difference somewhere.

<img width="641" alt="image" src="https://github.com/user-attachments/assets/59182af9-b5e2-4559-a2ef-c66dccfe1709">

Reply:
Going to mark this one as resolved and leave as something to check between the css in the template and that in the Apprenticeships dashboard.